recipes/larch-phylo/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "larch" %}
+{% set package_name = "larch-phylo" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ package_name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/matsengrp/larch.git
+  git_rev: conda-forge
```

Comment:
Please use an archive source with a verifiable hash. You can either make a release from this branch or get the GitHub download link from a specific commit hash. Otherwise, the recipe is not reproducible because tag pointing to "conda-forge" may be moved.